Roles/Neutral/Jester.cs

```diff
@@ -38,6 +39,8 @@ public override void SetupCustomOption()
         SunnyboyChance = IntegerOptionItem.Create(Id + 7, "SunnyboyChance", new(0, 100, 5), 0, TabGroup.NeutralRoles, false)
             .SetParent(CustomRoleSpawnChances[CustomRoles.Jester])
             .SetValueFormat(OptionFormat.Percent);
+        RevealJesterUponEjection = BooleanOptionItem.Create(Id + 8, GeneralOption.RevealUponEject, false, TabGroup.NeutralRoles, true)
+            .SetParent(CustomRoleSpawnChances[CustomRoles.Jester])
```

Comment:
need a ; at the end of this line
did u try to build before your pr

Reply:
Also setting disabled by defult
Change code to:
```
RevealJesterUponEjection = BooleanOptionItem.Create(Id + 8, GeneralOption.RevealUponEject, true, TabGroup.NeutralRoles, true)
            .SetParent(CustomRoleSpawnChances[CustomRoles.Jester]);
```

Reply:
> need a ; at the end of this line did u try to build before your pr

build? :|